server/helpers/express-utils.ts

```diff
@@ -125,6 +126,34 @@ function getCountVideos (req: express.Request) {
   return req.query.skipCount !== true
 }
 
+// helpers added in server.ts and used in subsequent controllers used
+const apiResponseHelpers = (req, res: express.Response, next = null) => {
+  res.fail = (options) => {
+    const { data, status, message, title, type, docs, instance } = {
+      data: null,
```

Comment:
I don't think you need this + check https://zaiste.net/posts/javascript-destructuring-assignment-default-values/ for default values